.github/workflows/test.yml

```
@@ -180,9 +180,9 @@ jobs:
     #  run: DEBUG=4 METAL=1 python -m pytest test/test_dtype.py
```

Maybe update this one as well? (it is commented out, but if someone uncomments it, they will introduce the same issue as this PR is trying to fix)

```suggestion
    #  run: DEBUG=4 METAL=1 python -m pytest -n=auto test/test_dtype.py
```